crypto/bn/bn_asm.c

```diff
@@ -197,7 +197,7 @@ void bn_sqr_words(BN_ULONG *r, const BN_ULONG *a, int n)
 
 BN_ULONG bn_div_words(BN_ULONG h, BN_ULONG l, BN_ULONG d)
 {
-    return ((BN_ULONG)(((((BN_ULLONG) h) << BN_BITS2) | l) / (BN_ULLONG) d));
+    return (BN_ULONG)(((((BN_ULLONG) h) << BN_BITS2) | l) / (BN_ULLONG) d);
```

Comment:
Hmm... I was under impression that this was specified in Coding Style, but apparently not. But if we follow other occurrences it looks like no space between cast and variable is preferred. I mean it feels that that there should not be spaces in `(BN_ULLONG) h` and `(BN_ULLONG) d`...

Reply:
Not easy decisions if the source itself uses both. Even counting occurrences is usually not indicative of the correct way.

On another issue, some macros do put arguments in parenthesis in the the definition, some not, I think the former is correct, the latter _may_ work depending on what's passed, f.e.:
`#define MYMACRO(a)  (a+1<1?1:0)` vs. `((a)+1<1?1:0)`